CHANGELOG.md

```diff
@@ -19,12 +19,16 @@ compatible with the updates.
 
 ### Changed
 
-- Refactored BSplines interpolation independetly of the backbone network and available
+- Uniformed local-net, global-net, u-net under a single u-net structure.
```

Comment:
not clear why the unifying is useful or advantageous

Reply:
It simplifies the code by a lot, and user can define their own U-Net by changing one component, like the up-sampling / down-sampling, or the conv block via overwriting one single function.